llvm/lib/Target/SystemZ/SystemZAsmPrinter.cpp

```diff
@@ -1296,6 +1298,41 @@ void SystemZAsmPrinter::emitPPA1(MCSymbol *FnEndSym) {
                                       4);
 }
 
+void SystemZAsmPrinter::emitGlobalVariable(const GlobalVariable *GV) {
+  if (TM.getTargetTriple().isOSzOS()) {
```

Comment:
if the class SystemZAsmPrinter is for ZOS only ? if it is,  do we need 
`if (TargetTriple.isOSzOS())` ? , please change to  `assert(TargetTriple.isOSzOS() &&...) `

Reply:
No, this class is used by Linux on Z as well.